systers_portal/community/forms.py

```diff
@@ -9,7 +9,30 @@
 from users.models import SystersUser
 
 
-class CommunityForm(ModelFormWithHelper):
+class AddCommunityForm(ModelFormWithHelper):
+    """ Form to create a new Community by admin. """
```

Comment:
If we go ahead with this, same we can use the organizer/user to submit the details of the new community as well. I see extra work in the use case of requester submitting through a SurveyMonkey form and then admin recreating all, using this form. It would be better if admin approves the details, submitted by requester using this form.

cc/ @MeepyMay.